### PR TITLE
addresing comments of pr15873

### PR DESCRIPTION
**Pull request by paolamvhz**
This requests addreses the changes suggested by srikanta in the pr 15873

### CODE REVIEW

File: sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java [hunk 1]
```diff
@@ -125,9 +125,15 @@
...
-        Objects.requireNonNull(phonePlanId, "'phonePlanId' cannot be null.");
+        if (locationOptions == null) {
+            Mono.error(new NullPointerException("'locationType' cannot be null."));
+        }
+        if (countryCode == null) {
+            Mono.error(new NullPointerException("'countryCode' cannot be null."));
+        }
+        if (phonePlanId == null) {
+            Mono.error(new NullPointerException("'phonePlanId' cannot be null."));
+        }
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java:136]: Like you did in the other method, move these inside the try block instead.. as a result the exception will be caught and return as monoError in the exception.. this needs to be done for all the methods in this client

**Review by ankitarorabit — CHANGES_REQUESTED** [review]
Added comments

File: sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java (hunk 1 above)
Comment (paolamvhz) [reply sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java]: Just surrond all the methods where there could be an exeception into a try

File: sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java [hunk 2]
```diff
@@ -127,14 +127,18 @@
...
         try {
+            if (locationOptions == null) {
+                Mono.error(new NullPointerException("'locationType' cannot be null."));
+            }
+            if (countryCode == null) {
+                Mono.error(new NullPointerException("'countryCode' cannot be null."));
+            }
+            if (phonePlanId == null) {
+                Mono.error(new NullPointerException("'phonePlanId' cannot be null."));
+            }
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java:139]: You don't need to put Mono.error inside the try block.. simply have the Objects.requireNonNull statements here and if they throw they will be caught and returned as Mono errors

File: sdk/communication/azure-communication-administration/swagger/README.md [hunk 3]
```diff
@@ -56,4 +56,4 @@ generate-client-as-impl: true
 custom-types-subpackage: models
 sync-methods: all
 context-client-method-parameter: true
-```
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/swagger/README.md:59]: What is the change in this file?

**Review by ankitarorabit — CHANGES_REQUESTED** [review]
Left a couple more comments

**Pull request closed without merging**